extras/indexing/src/main/java/org/apache/rya/indexing/accumulo/entity/EntityOptimizer.java

```diff
@@ -103,10 +104,8 @@ public void setConf(Configuration conf) {
                     eval = new AccumuloSelectivityEvalDAO(this.conf, ConfigUtils.getConnector(this.conf));
                     ((AccumuloSelectivityEvalDAO)eval).setRdfEvalDAO(new ProspectorServiceEvalStatsDAO(ConfigUtils.getConnector(this.conf), this.conf));
                     eval.init();
-                } catch (AccumuloException e) {
-                    e.printStackTrace();
-                } catch (AccumuloSecurityException e) {
-                    e.printStackTrace();
+                } catch (final AccumuloException | AccumuloSecurityException | TableExistsException e) {
+                    LOG.warn("A problem was encountered while setting the Configuration for the EntityOptimizer.", e);
```

Comment:
I don't know what else you'd say here. The chained exception will have the details.